python/ray/ml/tests/test_predictor.py

```diff
@@ -26,6 +31,18 @@ def test_serialization():
         ray.put(predictor)
 
 
+def test_scoring():
+    scorer = BatchScorer(DummyPredictor, Checkpoint.from_dict({"factor": 2.0}))
+
+    test_dataset = ray.data.from_items([1.0, 2.0, 3.0, 4.0])
+    assert scorer.score(test_dataset).to_pandas().to_numpy().squeeze().tolist() == [
```

Comment:
If looking for a more concise conversion + comparison, content should be assertable without this chain, something like:
```python
assert scorer.score(test_dataset).take() == [2.0, 4.0, 6.0, 8.0]
```

Reply:
This leads to 

```
E       AssertionError: assert [{'value': 8.0}, {'value': 6.0}, {'value': 4.0}, {'value': 2.0}] == [2.0, 4.0, 6.0, 8.0]
```

how do I get them as a series and not dicts?

Also, the order problem seems to remain, any insights on that?